### PR TITLE
Update elixir version to 1.4.1

### PR DESCRIPTION
**Pull request by Jesterovskiy**

### CODE REVIEW

File: frameworks/Elixir/phoenix/web/controllers/page_controller.ex [hunk 1]
```diff
@@ -5,17 +5,20 @@ defmodule Hello.PageController do
 
   def index(conn, _params) do
     conn
+    |> put_resp_content_type("application/json", nil)
```
Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: This should not be necessary. If it is, it is a Phoenix bug.

File: frameworks/Elixir/phoenix/web/controllers/page_controller.ex [hunk 2]
```diff
@@ -69,6 +74,7 @@ defmodule Hello.PageController do
 
   def plaintext(conn, _params) do
     conn
+    |> put_resp_content_type("text/plain", nil)
```
Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: Same here.

File: frameworks/Elixir/phoenix/web/controllers/page_controller.ex (hunk 1 above)
Comment (Jesterovskiy) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: @josevalim `utf-8` added by default to all `content-type`. And I see warning from FB. Maybe we can ignore it. What do you think?

Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: Weird, why isn't FB requiring a charset?

Comment (Jesterovskiy) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]:
```
Content encoding found in "text/plain; charset=utf-8" where "text/plain" is acceptable.
     Additional response bytes may negatively affect benchmark performance.
     See http://frameworkbenchmarks.readthedocs.org/en/latest/Project-Information/Framework-Tests/#specific-test-requirements
```
this warnings I see in Ruby frameworks too

Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: If we are going to explicitly set the content type, then I would use send_resp and give up on the `json` and `text` functions altogether. What do you think?

Comment (Jesterovskiy) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: Agree, good idea

File: frameworks/Elixir/phoenix/web/controllers/page_controller.ex [hunk 3]
```diff
@@ -31,7 +34,8 @@ defmodule Hello.PageController do
     end
 
     conn
-    |> json(Enum.map(1..q, fn _ -> Repo.get(World, :rand.uniform(10000)) end))
+    |> put_resp_content_type("application/json", nil)
+    |> send_resp(200, Poison.encode!(Enum.map(1..q, fn _ -> Repo.get(World, :rand.uniform(10000)) end)))
```
Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: Are we required per the spec to perform multiple queries? Or can we perform a single one?

Comment (Jesterovskiy) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: @josevalim  http://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#multiple-database-queries

**Comment by nbrady-techempower** [comment]
@Jesterovskiy Looks like Elixir/Cowboy had issue with this

File: frameworks/Elixir/phoenix/web/controllers/page_controller.ex (hunk 3 above)
Comment (josevalim) [inline frameworks/Elixir/phoenix/web/controllers/page_controller.ex]: Thanks!

**Comment by josevalim** [comment]
@Jesterovskiy looks good to me, thank you!

**Comment by Jesterovskiy** [comment]
@nbrady-techempower https://travis-ci.org/TechEmpower/FrameworkBenchmarks/builds/230251855 I think it fixed now. Check plz

**Comment by nbrady-techempower** [comment]
@Jesterovskiy looks good now, thank you!

**Pull request merged**